spec/filters/integration/elasticsearch_spec.rb

```diff
@@ -84,7 +84,9 @@
     end
 
     it "fails to register plugin" do
-      expect { plugin.register }.to raise_error Elasticsearch::Transport::Transport::Errors::Unauthorized
+      expect { plugin.register }.to raise_error elastic_ruby_v8_client_available? ?
```

Comment:
review note: no any logic related with this but this fixes one integration test failure.